src/index.js

```diff
@@ -251,27 +239,38 @@ var MaskedInput = React.createClass({
       onPaste: this._onPaste,
       [this._keyPressPropName()]: this._onKeyPress
     }
-  },
+  }
 
   focus() {
     this.input.focus()
-  },
+  }
 
   blur() {
     this.input.blur()
-  },
+  }
 
   render() {
-    var ref = r => this.input = r
+    var ref = r => { this.input = r }
```

Comment:
es5 to es6 was not my purpose, I just wanted to avoid React warnings. I think it will be better to do these things in other PR, cause this PR relates to React.createClass only